integration-tests/src/test/java/com/datastax/oss/driver/mapper/StatementAttributesIT.java

```diff
@@ -78,6 +75,88 @@ public void setup() {
 
   @Test
   public void should_honor_runtime_attributes_insert() {
+    primeInsertQuery();
+    CqlSession session = SessionUtils.newSession(simulacronRule);
+    InventoryMapper inventoryMapper =
+        new StatementAttributesIT_InventoryMapperBuilder(session).build();
+    SimpleDao simpleDao = inventoryMapper.simpleDao(sessionRule.keyspace());
+    StatementAttributes attributes = buildRunTimeAttributes();
+    simulacronRule.cluster().clearLogs();
+    simpleDao.save(simple, attributes);
+    ClusterQueryLogReport report = simulacronRule.cluster().getLogs();
+    validateQueryOptions(report.getQueryLogs().get(0), true);
+  }
+
+  @Test
+  public void should_honor_attributes_insert() {
```

Comment:
maybe you could change the name of this test to: `should_honor_annotation_attributes_insert` (ant other tests that use: `@WithStatementAttributes`) for better readability?